README.md

```diff
@@ -1,6 +1,6 @@
-![Logo](https://raw.githubusercontent.com/idealista/opengrok-role/master/logo.gif)
+![Logo](https://raw.githubusercontent.com/idealista/opengrok_role/master/logo.gif)
 
-[![Build Status](https://travis-ci.org/idealista/opengrok-role.png)](https://travis-ci.org/idealista/opengrok-role)
+[![Build Status](https://travis-ci.org/idealista/opengrok_role.png)](https://travis-ci.org/idealista/opengrok_role)
```

Comment:
Now we're using travis-ci.com.

travis-ci.org/* is just redirecting to travis-ci.com/*